### ISSUE TITLE
Missing files from labels.zip in BaeduNetdisk

### ISSUE THREAD

**Issue by gmindflow** [18979705623/HSPR#2]
Hello, thank you for open sourcing your code of the great work!
I am trying to reproduce the results of you work but have a small problem.

Two files that are needed in running the code `sh pretrain_auxiliary_src/run_reverie.sh`, which are [sub_region_label.txt ](https://github.com/18979705623/HSPR/blob/fb233d271e2e2d9ce47e011af90cdeb6fd3629e2/pretrain_auxiliary_src/suat/reg_sub_area.py#L10) and [candidate_label.json](https://github.com/18979705623/HSPR/blob/fb233d271e2e2d9ce47e011af90cdeb6fd3629e2/pretrain_auxiliary_src/suat/reg_sub_area.py#L17) seems to be missing from the labels.zip file that can be downloaded from BaeduNetdisk.

Could you please upload these files on the Netdisk?
Thank you.

**Comment by XieZilongAI** [comment]
Thank you very much for your attention.  During the pre-training stage, the generation of these two files is facilitated by the scene understanding auxiliary tasks.  The candidate_label. json file is generated by the get_candidate.py script located in the suat folder.  The generation of the sub_region_label.txt file is handled by the reg_sub_area.py script.
Thank you.

**Comment by gmindflow** [comment]
Thank you for your feedback!

I believe people would find it useful if it is included in the README as well.
Thanks.
